lib/ReactViews/Credits/MapCredits/MapCredits.tsx

```diff
@@ -0,0 +1,60 @@
+import { reaction } from "mobx";
+import { observer } from "mobx-react";
+import React, { FC, useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { useDataAttributionContext } from "../../../ReactContexts/DataAttributionContext";
+import { Credits, Spacer } from "../Credits";
+import { CreditsContainer } from "../CreditsContainer";
+import { DataAttributionModal } from "../DataAttribution/DataAttributionModal";
+import { TerriaLogo } from "../TerriaLogo";
+import { MapCreditLogo } from "./MapCreditLogo";
+import { IMapCreditsProps } from "./MapCredits.props";
+
+export const MapCredits: FC<IMapCreditsProps> = observer(
```

Comment:
`import React from "react";` should not be used in react 17 and later it will deprecated in future version of React. We still can't upgrade to React 17 or 18, see https://github.com/TerriaJS/terriajs/pull/5549/files#issuecomment-922506060 for details, but IMHO it would be good to start adopting new pattern

Reply:
Yes I agree with you, thanks for the warning.